sdk/src/schema/preset.ts

```diff
@@ -0,0 +1,23 @@
+import type { MergeCoreCollection } from '../index.js';
+import type { DirectusRole } from './role.js';
+import type { DirectusUser } from './user.js';
+
+export type DirectusPreset<Schema extends object> = MergeCoreCollection<
+	Schema,
+	'directus_presets',
+	{
+		id: number;
+		bookmark: string | null;
+		user: DirectusUser<Schema> | string | null;
+		role: DirectusRole<Schema> | string | null;
```

Comment:
Should be just as it only accepts / returns the UUIDs
```suggestion
		user: string | null;
		role: string | null;
```
or maybe
```suggestion
		user: DirectusUser<Schema>['id'] | null;
		role: DirectusRole<Schema>['id'] | null;
```

Reply:
I dont think that is true 🤔 the `/presets` endpoint does accept the `fields` parameter so `?fields=*,user.*` will fetch you a complete user object

Reply:
Oh, I see 👍 But not for the request, right?

Reply:
Discussed on Discord, types are correct ✔️